Libraries/pbxbuild/Sources/WorkspaceContext.cpp

```diff
@@ -143,7 +152,38 @@ LoadWorkspaceProjects(Filesystem const *filesystem, std::vector<pbxproj::PBX::Pr
 }
 
 static void
-LoadNestedProjects(Filesystem const *filesystem, std::vector<pbxproj::PBX::Project::shared_ptr> *projects, pbxsetting::Environment const &baseEnvironment, std::vector<pbxproj::PBX::Project::shared_ptr> const &rootProjects)
+LoadConfigurationFiles(
+    Filesystem const *filesystem,
+    std::unordered_map<pbxproj::XC::BuildConfiguration::shared_ptr, pbxsetting::XC::Config> *configs,
+    pbxsetting::Environment const &environment,
+    pbxproj::XC::ConfigurationList::shared_ptr const &configurationList)
+{
+    if (configurationList == nullptr) {
```

Comment:
Isn't configurationList a reference?  I didn't think it was reasonable to check whether such types were null.